modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/quote-navbar/index.html

```diff
@@ -1,27 +1,60 @@
-[#assign
-	publicFriendlyURL = themeDisplay.getPathFriendlyURLPublic() + themeDisplay.getSiteGroup().getFriendlyURL()
-/]
+[#assign publicFriendlyURL = themeDisplay.getPathFriendlyURLPublic() +
+themeDisplay.getSiteGroup().getFriendlyURL() /]
 
 <nav class="navbar navbar-expand-lg navbar-light quote-site-navbar">
 	<div class="container">
 		<div class="flex-grow-1 navbar-brand">
 			<a class="navbar-brand" href="${publicFriendlyURL}">
-				<img alt="" class="align-top d-inline-block navbar-logo" data-lfr-editable-id="image-logo" data-lfr-editable-type="image" loading="lazy" src="[resources:raylife_logo.svg]" />
+			<img alt="" class="align-top d-inline-block navbar-logo" data-lfr-editable-id="image-logo" data-lfr-editable-type="image" loading="lazy" src="[resources:raylife_logo.svg]" />
 
-				<span class="brand-subtext font-weight-bolder text-brand-secondary text-paragraph-sm">Commercial</span>
+			<span class="brand-subtext font-weight-bolder text-brand-secondary text-paragraph-sm">Commercial</span>
 			</a>
 		</div>
 
-		<div class="pb-2 pl-2 pr-2 pt-2 rounded-pill shadow-lg">
+		<div class="m-2 p-2 rounded-pill shadow-lg" id="navbar-contact-phone">
 			<span class="text-neutral-7 text-link-sm">(909) 987-6543</span>
 		</div>
 
-		<div class="pb-2 pl-2 pr-2 pt-2 rounded-pill shadow-lg">
+		<div class="bg-neutral-0 mr-2 pb-2 pl-2 pr-2 pt-1 rounded-pill shadow-lg">
 			<img alt="" src="[resources:help.svg]" />
 		</div>
 
-		<div class="pb-2 pl-2 pr-2 pt-2 rounded-pill shadow-lg">
-			<img alt="" src="[resources:voice-chat.svg]" />
+		<div class="bg-neutral-0 mr-2 pb-2 pl-2 pr-2 pt-1 rounded-pill shadow-lg voice-chat-icon">
+			<img alt="" src="[resources:voice_chat.svg]" />
+		</div>
+
+		<div class="menu-drop">
+		<button aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation" class="ghost navbar-toggler raylife-navbar-button" data-target="#navbarSupportedContent" data-toggle="collapse" type="button">
+				<div class="align-items-center d-flex justify-content-between">
+				<span class="font-weight-bolder mr-1 paragraph-sm">MENU</span>
+					<span class="background-icon-grid" id="button-men-icon">
```

Comment:
Try to specify the ```id``` better. I think it should be ```button-menu-icon``` instead ```button-men-icon```. Please modify this information in all files that it is used.